### ISSUE TITLE
French-Wilson script

### ISSUE THREAD

**Issue by tjlane** [rs-station/rs-booster#51]
@kmdalton what do you think? If you guys have a good implementation, a simple CLI utility to turn intensities into SFAs would be great.

I'd love to not have to use `phenix.reflection_file_converter`...

**Comment by tjlane** [comment]
And I should say -- I volunteer myself for this task.

**Comment by JBGreisman** [comment]
seems reasonable -- we have a French-Wilson implementation in `reciprocalspaceship`: https://github.com/rs-station/reciprocalspaceship/blob/cbf30c1c8438326baddf6a9898c2f5b5be3f7b43/reciprocalspaceship/algorithms/scale_merged_intensities.py#L178

We also currently use this in `rs-booster` as an internal IO convenience, so I agree it would make sense to have a standalone CLI if you think that would be useful: https://github.com/rs-station/rs-booster/blob/741c2bba9dbad3b0fd704bba68cfb7da41d2f1b3/rsbooster/utils/io.py#L40-L46

**Comment by kmdalton** [comment]
Yeah this makes sense to me. It should be relatively quick to implement. Let me know if you need help.

**Comment by DHekstra** [comment]
here's a first attempt:

[french_wilson.py.txt](https://github.com/user-attachments/files/21245726/french_wilson.py.txt)
